test/compare-acorn-es2020/nullish-coalescing.expected.js

```diff
@@ -0,0 +1,12 @@
+a ?? b;
+a ?? b ?? c;
+a | b ?? c | d;
+a ?? b ? c : d;
```

Comment:
Might as well be a little more thorough:

```suggestion
a ?? b ? c : d;
a ? b ?? c : d;
a ? b : c ?? d;
a => b ?? c;
```

Reply:
Thank you https://github.com/estools/escodegen/pull/417/commits/5646f113ae78eca9046da1b8e4ea20d4baec87ad